osu.Game/Screens/Play/SongProgressInfo.cs

```diff
@@ -85,8 +85,8 @@ protected override void Update()
 
             if (currentSecond != previousSecond && songCurrentTime < songLength)
             {
-                timeCurrent.Text = TimeSpan.FromSeconds(currentSecond).ToString(songCurrentTime < 0 ? @"\-m\:ss" : @"m\:ss");
-                timeLeft.Text = TimeSpan.FromMilliseconds(endTime - AudioClock.CurrentTime).ToString(@"\-m\:ss");
+                timeCurrent.Text = TimeSpan.FromSeconds(currentSecond).ToString(songCurrentTime < 0 ? @"\-h\:mm\:ss" : @"h\:mm\:ss");
```

Comment:
I think you should only be showing hours if the map goes for over an hour.  `0:02:30` looks horrible, when the majority of maps are going to be under 10 minutes.

Reply:
0:02:30 looks normal and fine to me, I've seen the h:mm:ss format in a lot of timers in and out of video games. But, I see your point.

Reply:
That initial `0:0` will be redundant for anything but marathons and unrankables.  Sometimes less information is clearer.

Reply:
Alright, understood.

Reply:
Personally i'd prefer it to keep counting in minutes, not hours.